partner_identification_dea/models/res_partner_id_category.py

```diff
@@ -0,0 +1,16 @@
+# Copyright (C) 2021 Open Source Integrators
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class ResPartnerIdCategory(models.Model):
+    _inherit = "res.partner.id_category"
+
+    dea_number = fields.Char(
+        string="DEA #",
+    )
+
+    medical_license = fields.Char(
+        string="Medical License",
+    )
```

Comment:
Do not add those fields since. Those information will be put on **res.partner.id_number** as **name** and **category_id**